clojure_liberator/src/clojure_liberator/core.clj

```diff
@@ -16,23 +16,26 @@
   (-write [date out]
     (json/-write (fmt/unparse formatter date) out)))
 
-(defn- make-json-api-item [type-name value]
-  {:id         (str (:id value))
-   :type       type-name
-   :attributes (dissoc value :id)})
-
-(defn- update-vals [map vals f]
-  (reduce #(update-in %1 [%2] f) map vals))
+(defn- update-vals [map fields transformation]
+  (reduce #(update-in %1 [%2] transformation) map fields))
 
 (defn- parse-date [value]
   (fmt/parse formatter value))
 
 (defn- replace-dates [input date-fields]
-  (update-vals input (filter #(input %) date-fields) #(parse-date %)))
+  "Takes the specified collection of date-fields and converts them from ISO format to joda DateTime instances.
```

Comment:
I've seen Clojure comments with `(comment ... )` or `;;`, but not this! This is legit?